api/response/schema.py

```diff
@@ -71,5 +71,36 @@ def resolve_get_room_response(self, info, **kwargs):
         return room_feedback
 
 
+class HandleRoomResponse(graphene.Mutation):
+    """
+        Returns payload on marking or unmarking
+        a response as resolved
+    """
+    class Arguments:
+        response_id = graphene.Int()
+
+    room_response = graphene.Field(Response)
+
+    @Auth.user_roles('Admin')
+    def mutate(self, info, response_id, **kwargs):
+        query_responses = Response.get_query(info)
+        room_response = query_responses.filter(
+            ResponseModel.id == response_id).first()
+        if not room_response:
+            raise GraphQLError("Response does not exist")
+        if room_response.resolved:
+            room_response.resolved = False
+            room_response.save()
+        else:
+            room_response.resolved = True
+            room_response.save()
+        return HandleRoomResponse(room_response=room_response)
+
+
 class Mutation(graphene.ObjectType):
     create_response = CreateResponse.Field()
+    handle_room_response = HandleRoomResponse.Field(
```

Comment:
Change this to `resolve_room_response`

Reply:
fix failing tests too

Reply:
I think you should re-run the workflow on CircleCI. @nakatuddesuzan

Reply:
@verenceLola , I had to update my the down revision as a PR with an Alembic file was merged

Reply:
@Bonifase , this has been worked on